docs/installation.rst

```diff
@@ -9,7 +9,39 @@ There are two ways to build/install this package.
 Install as Python package in a conda environment
 ----------------------------------------------------
 
-Please refer to the `repository readme`_ for instructions.
+Prerequisites: Install Anaconda or Miniconda
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If not already done, install ``conda`` (Miniconda is sufficient).  To do so, see the
+`official documentation <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`_.
+
+We tested with conda version 4.8.3.
+
+Installing the trifinger_simulation package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Clone this repo and cd into it::
+
+       git clone https://gitlab.is.tue.mpg.de/robotics/trifinger_simulation.git
+       cd trifinger_simulation
+
+2. Set up the conda env::
+
+       export PYTHONNOUSERSITE=True
```

Comment:
I assume you removed the note on `PYTHONNOUSERSITE` below because it is not so important in a non-challenge context?  In this case, could it be removed here as well?

Reply:
Okay